### PR TITLE
add OCA to author

### PR DESCRIPTION
**Pull request by OSguard**
to all modules

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by tafaRU** [comment]
thanks :+1:

File: web_widget_x2many_2d_matrix/__openerp__.py [hunk 1]
```diff
@@ -21,7 +21,8 @@
 {
     "name": "2D matrix for x2many fields",
     "version": "1.0",
-    "author": "Therp BV",
+    "author": "Therp BV, "
+              "Odoo Community Association (OCA)",,
     "license": "AGPL-3",
```
Comment (legalsylvain) [inline web_widget_x2many_2d_matrix/__openerp__.py:26]: there is a typo. (2 coma).
Otherwise. thanks :+1:

**Comment by legalsylvain** [comment]
:+1:. Thanks

**Pull request merged**